### PR TITLE
Align configuration with qontract-reconcile

### PR DESCRIPTION
**Pull request by janboll**
This aligns the integration configuration with the configuration used in https://github.com/app-sre/qontract-reconcile

It also adds a template that can be used to deploy integrations using a saas file

https://issues.redhat.com/browse/APPSRE-6816

### CODE REVIEW

File: internal/account_notifier.go [hunk 1]
```diff
@@ -145,6 +145,7 @@ func (n *AccountNotifier) CurrentState(ctx context.Context, ri *ResourceInventor
 		if err != nil {
 			return errors.Wrap(err, fmt.Sprintf("Error while reading secret %s", secretPath))
 		}
+		fmt.Println(secret)
```
Comment (geoberle) [inline internal/account_notifier.go:148]: :)

**Review by geoberle — COMMENTED** [review]
nice

File: internal/account_notifier.go (hunk 1 above)
Comment (janboll) [reply internal/account_notifier.go:148]: Ups, thanks for spotting that.

**Review by patjlm — APPROVED** [review]

File: openshift/deployment-template.yml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+          imagePullPolicy: ${INTERNAL_CERTIFICATES_IMAGE_PULL_POLICY}
+          command: ["/bin/sh", "-c"]
+          args:
+          - |
+            cp -r /etc/pki/. /tmp/etc-pki/
+          volumeMounts:
+          - name: internal-certificates
+            mountPath: /tmp/etc-pki/
+        containers:
+        - image: ${IMAGE}:${IMAGE_TAG}
```
Comment (patjlm) [inline openshift/deployment-template.yml:45]: how is slack logging performed? directly within go logger ? no need for a sidecar fluentd?

Comment (janboll) [reply openshift/deployment-template.yml:45]: Would opt for side car deployment, as in qontract-reconcile if it was required. However I'm reluctant to add all these features to the template. I'd rather go for integration go-qontract-reconcile with deployment manager.

**Pull request merged**